pants

```diff
@@ -193,4 +205,11 @@ pants_dir="$(bootstrap_pants "${pants_version}" "${python}")"
 # See https://blog.phusion.nl/2017/10/13/why-ruby-app-servers-break-on-macos-high-sierra-and-what-can-be-done-about-it/
 export no_proxy='*'
 
-exec "${pants_dir}/bin/python" "${pants_dir}/bin/pants" "$@"
+if [[ -z "${pants_config_file}" ]]; then
+  pants_config_file_flag="";
+else
+  pants_config_file_flag=--pants-config-files="[\"${PWD}/${pants_config_file}\"]";
```

Comment:
The effect of this is that if `PANTS_CONFIG_FILES` is set in the environment, this flag will be applied last, leading to something like:
```
pants_config_files = ['/Users/example/repo/pants.toml', '/Users/example/repo/pants.ci.toml', '/Users/example/repo/pants.toml'] (from FLAG from env var PANTS_CONFIG_FILES)
```
... which in this case causes some of the values in `pants.ci.toml` to not to be applied.